dataconnect/lib/profile.dart

```diff
@@ -31,13 +31,6 @@ class _ProfileState extends State<Profile>
   @override
   void initState() {
     super.initState();
-    Auth.getCurrentUser().then((user) {
```

Comment:
Why was this deleted?

Reply:
Fixed! I forgot to add the authguard there. I was thinking the user came from the future